aws/protocol/rest/header.go

```diff
@@ -43,17 +43,11 @@ func newHeaderValue(header http.Header, key string, append bool) HeaderValue {
 }
```

Comment:
Do we need `newHeaderValue` constructor? It isn't exported and we can just use `HeaderValue` directly on line 31.

Reply:
This was just to keep construction of the header consistent as the structure is created in a number of spots.